src/s_stuff.h

```diff
@@ -136,7 +136,6 @@ typedef struct _audiosettings
 #define DEFAULTSRATE 48000
 #ifdef _WIN32
```

Comment:
since we are at it, instead of nested `#ifdef`s I think it's better to do the following:

```
#if defined(_WIN32)
...
#elif defined(__APPLE__)
...
#else
...
#endif